tests/test_scheduler.py

```diff
@@ -44,12 +87,13 @@ def test_scheduler_prop_not_defined_raises_key_error():
 
 def test_scheduler_known_args():
 
-    expected = """#SBATCH --extra-flags
-#SBATCH --extra_arg=12345
+    expected = """#SBATCH --nodes=12
 #SBATCH --job-name=abcd
-#SBATCH scheduler=slurm"""
+#SBATCH --extra_arg=12345
+#SBATCH --extra-flags"""
```

Comment:
Shouldn't this test fail since it does not have all the required attributes of `queue`, account`, `tasks_per_node`?

Reply:
Fixed in most recent commit.